src/QsCompiler/LanguageServer/FileSystemWatcher.cs

```diff
@@ -231,13 +258,18 @@ private void RecurDeleted(string fullPath)
 
         private void RecurChanged(string fullPath)
         {
-            if (this.WatchedDirectories.TryGetValue(new Uri(fullPath), out var _)) { } // nothing to do here
-            else this.OnChangedFile(fullPath);
+            if (this.watchedDirectories.TryGetValue(new Uri(fullPath), out _))
+            {
+                // nothing to do here
+            }
+            else
+            {
+                this.OnChangedFile(fullPath);
+            }
```

Comment:
The condition here could be negated, removing the need for an empty code block. Although, doing changes like this might be out of scope for the PR.

Reply:
I agree, but yes, I'd like to avoid doing any changes in this PR that aren't strictly necessary - it's too tempting to go overboard. For example, `TryGetValue` could also be replaced with `ContainsKey` here. :)